docs/nlu/components.rst

```diff
@@ -1458,204 +1500,161 @@ DIETClassifier
-            "loss_type": "softmax"
-            # Number of top actions to normalize scores for loss type 'softmax'.
-            # Set to 0 to turn off normalization.
-            "ranking_length": 10
-            # Indicates how similar the algorithm should try to make embedding vectors
-            # for correct labels.
-            # Should be 0.0 < ... < 1.0 for 'cosine' similarity type.
-            "maximum_positive_similarity": 0.8
-            # Maximum negative similarity for incorrect labels.
-            # Should be -1.0 < ... < 1.0 for 'cosine' similarity type.
-            "maximum_negative_similarity": -0.4
-            # If 'True' the algorithm only minimizes maximum similarity over
-            # incorrect intent labels, used only if 'loss_type' is set to 'margin'.
-            "use_maximum_negative_similarity": True
-            # Scale loss inverse proportionally to confidence of correct prediction
-            "scale_loss": True
-            # ## Regularization parameters
-            # The scale of regularization
-            "regularization_constant": 0.002
-            # The scale of how important is to minimize the maximum similarity
-            # between embeddings of different labels.
-            "negative_margin_scale": 0.8
-            # Sparsity of the weights in dense layers
-            "weight_sparsity": 0.8
-            # Dropout rate for encoder
-            "drop_rate": 0.2
-            # Dropout rate for attention
-            "drop_rate_attention": 0
-            # If 'True' apply dropout to sparse tensors
-            "use_sparse_input_dropout": True
-            # ## Evaluation parameters
-            # How often calculate validation accuracy.
-            # Small values may hurt performance, e.g. model accuracy.
-            "evaluate_every_number_of_epochs": 20
-            # How many examples to use for hold out validation set
-            # Large values may hurt performance, e.g. model accuracy.
-            "evaluate_on_number_of_examples": 0
-            # ## Model config
-            # If 'True' intent classification is trained and intent predicted.
-            "intent_classification": True
-            # If 'True' named entity recognition is trained and entities predicted.
-            "entity_recognition": True
-            # If 'True' random tokens of the input message will be masked and the model
-            # should predict those tokens.
-            "use_masked_language_model": False
-            # 'BILOU_flag' determines whether to use BILOU tagging or not.
-            # If set to 'True' labelling is more rigorous, however more
-            # examples per entity are required.
-            # Rule of thumb: you should have more than 100 examples per entity.
-            "BILOU_flag": True
+    You can define a number of hyperparameters to adapt the model.
+    If you want to adapt your model, start by modifying the following parameters:
```

Comment:
@Ghostvv What parameters do you think users should modify first when adapting the model?

Reply:
epochs

Reply:
number of transformer layers,
transformer_size,
hidden_layers_sizes,
embedding_dimension

Reply:
I'm not sure

Reply:
yeah, also not sure, will add 
transformer_size,
hidden_layers_sizes,
embedding_dimension
thanks

Reply:
let's also put `number of transformer layers` in there, that's the parameter that people probably are most familiar with

Reply:
is already in there, just updated